.github/workflows/ci-cd.yml

```
@@ -146,16 +156,26 @@ jobs:
 
   docker:
     name: Build and Push Docker Image
-    needs: [test, auto-tag]
+    needs: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)